docs/setup/jetson.md

```diff
@@ -81,49 +247,28 @@ It is highly recommended to read through the getting started document for your N
     !!! warning
         Be sure that the [USB/BLE toggle on the robot's adapter board](../../hw/electrical/#adapter-board-overview) is set to the USB position.
 
+3. Check if you can ping Create 3
+
+        ping 192.168.186.2
+
     !!! note
         If you ever want to stop using the Jetson device for Create® 3 and re-enable the original USB Device Mode feature (so that you can connect to your Windows PC in headless style), you can simply remove the flag file, or execute the following.
 
         sudo mv /opt/nvidia/l4t-usb-device-mode/IP_ADDRESS_FOR_CREATE3_ROBOT.conf /opt/nvidia/l4t-usb-device-mode/IP_ADDRESS_FOR_CREATE3_ROBOT.conf.bak
 
+#### Test the USB link inside the container
 
-### Start ROS 2 Galactic container
+Perform the followings in the ROS 2 container.
 
-The recommended way to run ROS2 on Jetson is to use pre-built Docker container images.
+1. Set the `ROS_DOMAIN_ID` environmental variable 
 
-1. Enable NVIDIA Container Runtime with Docker integration, if this is the first time running containers on Jetson, to allow access to GPU from containers.
+    Once you are in container;
```

Comment:
should this be a colon and not a semicolon?

Reply:
[9f8494a](https://github.com/iRobotEducation/create3_docs/pull/385/commits/9f8494a47bf1367bb3174178a2d721b18b730202)